ci/script.sh

```diff
@@ -20,12 +20,12 @@ case $TARGET in
 		sudo apt-get -y update
```

Comment:
I think this script is not used anymore and could be deleted.

Reply:
Uh, not sure whether I should be doing that deletion in this PR though...